daffodil-udf/src/test/java/org/badudfs/annotations/StringFunctions/Compare.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.badudfs.annotations.StringFunctions;
+
+import java.io.Serializable;
+
+import org.apache.daffodil.udf.FunctionClassInfo;
+
+@FunctionClassInfo(
```

Comment:
Might put some documentation here about why this is bad. I could imagine someone finding this as an example and not realizing it's a negative test and things not working for them.

Reply:
Done